.github/workflows/mavenBuild.yml

```
@@ -68,6 +73,13 @@ jobs:
       uses: stCarolas/setup-maven@d6af6abeda15e98926a57b5aa970a96bb37f97d1 # v5
       with:
         maven-version: ${{ inputs.mavenVersion }}
+
+    - name: Install Maven parent locally
+      uses: HannesWell/eclipse.platform.releng.aggregator/.github/actions/install-maven-parent-action@gh-workflow-quickfix
```

This and the other two calls have to be replaced by the final location before this can be submitted.
```suggestion
      uses: eclipse-platform/eclipse.platform.releng.aggregator/.github/actions/install-maven-parent-action@master
```

---

Done.